modules/stvBidAdapter.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+```
+Module Name: STV Video Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid@dspx.tv
+```
+
+# Description
+
+STV video adapter for Prebid.js 1.x
+
+# Parameters
+```
+    var adUnits = [
+        {
+            // video settings
+            code: 'video-obj',
+            mediaTypes: {
+                video: {
+                    context: 'instream',
+                    playerSize: [640, 480]
+                }
+            },
+            bids: [
+                {
+                    bidder: "stv",
+                    params: {
+                        placement: "", // placement ID of inventory with STV
```

Comment:
I am not able to test with these params.

Can you please provide a test `placement` which will return a video ad?

Reply:
@robertrmartinez Could You help me to solve one problem? The STV adapter already merged into master (https://github.com/prebid/Prebid.js/pull/3737) but I see its commit in this branch. I don't know how it got into this branch.

Our GitLab changes:

![image](https://user-images.githubusercontent.com/48312668/65540598-bfb68900-df1c-11e9-8113-60f4d33abb88.png)

GitHub commits: 

![image](https://user-images.githubusercontent.com/48312668/65540659-dbba2a80-df1c-11e9-9cdb-6d874d16799d.png)


We have a `Mirror a repository` settings in our GitLab application. What could happen and how to fix it?